plugins/inmemoryscanner/Readme.md

```diff
@@ -144,7 +141,6 @@ plugin_system:
       signature_file: /usr/local/share/inmemsigs/sigs.sig
       dump_memory: false
       scan_all_regions: false
-      maximum_scan_size: 52428800
```

Comment:
Looks like this discussion is still valid after all. `vmi_mmap_guest` appears to have a significant performance impact when very large (but probably very empty) regions are involved since a v2p lookup is performed for every single 4kb page.